Fluid/DefaultFluidParser.cs

```diff
@@ -893,7 +901,28 @@ public static FilterArgument BuildFilterArgument(ParseTreeNode node)
             return new FilterArgument(identifier, term);
         }
 
+        private static AssignStatement BuildIncludeAssignStatement(ParseTreeNode tag)
+        {
+            var identifier = tag.ChildNodes[0].Token.ValueString;
+            var value = BuildTermExpression(tag.ChildNodes[1]);
+
+            return new AssignStatement(identifier, value);
+        }
+
         #endregion
+
+        private static void Traverse(ParseTreeNode tag, ref IList<AssignStatement> statements)
```

Comment:
Why `ref`?

Reply:
What is recursive in include assignments?

Reply:
> Why ref?

The `ref` needed to keep track the `AssignStatement`s during the traversing, I could make as global field instead and remove from here .. I think this a better choice
> What is recursive in include assignments?

The recursive needed to visit all the node in the assignment tree